net/ethtool/Makefile

```diff
@@ -37,6 +39,15 @@ define Package/ethtool/description
  network interface
 endef
 
+define Package/ethtool/config
+	source "$(SOURCE)/Config.in
+endef
+
+define Build/Configure
+	$(call Build/Configure/Default,\
+	$(if $(CONFIG_ETHTOOL_PRETTY_DUMP),--enable,--disable)-pretty-dump)
+endef
```

Comment:
Is there a reason you used a Build/Configure section instead of just adding the option to CONFIGURE_ARGS?